app/appmessage/rpc_submit_block.go

```diff
@@ -19,11 +19,32 @@ func NewSubmitBlockRequestMessage(block *MsgBlock) *SubmitBlockRequestMessage {
 	}
 }
 
+// RejectReason describes the reason why a block sent by SubmitBlock was rejected
+type RejectReason byte
+
+// RejectReason constants
+const (
+	RejectReasonNone RejectReason = iota
```

Comment:
I think it's better not to use iota if we use the hardcoded numbers on rpc.proto